app/views/notify_user/metric_threshold_crossed.text.haml

```diff
@@ -0,0 +1,28 @@
+Client Metric Thresholds Crossed
+
+The following clients have crossed metric thresholds on #{@calculation_date.strftime('%B %d, %Y')}:
+
+- @crossings.each do |metric_name, snapshot_info|
+  \
+  #{metric_name}
+  = '-' * metric_name.length
+  #{snapshot_info[:total_count]} #{'client'.pluralize(snapshot_info[:total_count])} crossed the threshold for this metric.
+  - if snapshot_info[:truncated]
+    (Showing first 50)
+
+  - snapshot_info[:data].each do |data|
+    \
+    Client ##{data[:entity_id]}: #{client_url(id: data[:entity_id])}
+    - if data[:previous_value]
+      Previous Value: #{data[:previous_value]}
+      New Value: #{data[:current_value]}
+      - change = data[:current_value] - data[:previous_value]
+      - sign = change > 0 ? '+' : ''
+      Change: #{sign}#{change}
+    - else
+      New Value: #{data[:current_value]}
+      (New baseline)
+
+\
+You are receiving this notification because you are subscribed to client metric alerts.
+To manage your alert subscriptions, contact your administrator.
```

Comment:
maybe this is okay, and cheaper simpler. Generally would better to allow users to opt-out of receiving any automated email (supports 'unsubscribe' link in mail footer).